lib/tasks/db.rake

```diff
@@ -0,0 +1,14 @@
+namespace :db do
```

Comment:
Simple rake task adding:
`rails db:drop:connections` to make dropping and recreating a postgres database easier